docs/about/release_notes.rst

```diff
@@ -27,6 +27,99 @@ Release Notes
  Miscellaneous:
    -
 
+
+Version 8.0
+-----------
+
+In this version of ERT we have clean up a lot of unused analysis keywords. Some have been deprecated,
+while others that were previously deprecated have now been removed. If they are present in your
+configuration, you will get a message in the suggestor at startup:
+
+.. image:: version-8.0-suggestor.png
+
+We encourage everyone to keep their configuration updated and remove and replace deprecated keywords.
+
+Another big change this release is the change to a streaming algorithm. The streaming algorithm
```

Comment:
Maybe define `streaming algorithm` a bit.

```suggestion
Another big change this release is the change to a streaming algorithm, which updates a single parameter group at a time instead of updating all at once. This means ERT only needs to hold one parameter group in memory at a time ,which significantly reduces memory usage when doing updates, without any effect to the result of the update.
```